magento/magento2
# 'On-shopping-cart-in-summary-shipping-options-missalinged' :: add cha…

…nges for On shopping cart in summary shipping options missalinged





### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#20131: On shopping cart in summary shipping options missalinged
2. ...

### Manual testing scenarios (*)

1. Ad product to cart
2. and go to cart page, check summary block for shipping options

### Contribution checklist (*)
